app/src/main/java/org/akvo/flow/util/OldHttpUtil.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2010-2017 Stichting Akvo (Akvo Foundation)
+ *
+ * This file is part of Akvo FLOW.
+ *
+ * Akvo FLOW is free software: you can redistribute it and modify it under the terms of
+ * the GNU Affero General Public License (AGPL) as published by the Free Software Foundation, either version 3 of the License or any later version.
+ *
+ * Akvo FLOW is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License included below for more details.
+ *
+ * The full license text can also be seen at <http://www.gnu.org/licenses/agpl.html>.
+ *
+ */
+
+package org.akvo.flow.util;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import org.akvo.flow.exception.HttpException;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Map;
+import java.util.Map.Entry;
+
+/**
+ * Simple utility to make http calls and read the responses
+ *
+ * @author Christopher Fagiani
+ */
+public class OldHttpUtil {
+
+    private static final String TAG = OldHttpUtil.class.getSimpleName();
+    private static final int BUFFER_SIZE = 8192;
+
+    @NonNull
+    public static String httpGet(String url) throws IOException {
+        HttpURLConnection conn = (HttpURLConnection) (new URL(url).openConnection());
+        final long t0 = System.currentTimeMillis();
+
+        try {
+            int status = getStatusCode(conn);
+            if (status != HttpURLConnection.HTTP_OK) {
+                throw new HttpException(conn.getResponseMessage(), status);
+            }
+            InputStream in = new BufferedInputStream(conn.getInputStream());
+            String response = readStream(in);
+            Log.d(TAG, url + ": " + (System.currentTimeMillis() - t0) + " ms");
```

Comment:
In the recent version of the develop I have included the timber library which is a nice tool for logging. With Timber, your code would look like this: Timber.d("%s: %f ms", url, (System.currentTimeMillis() - t0))